src/migrator/sync.ts

```diff
@@ -52,6 +54,14 @@ export const run = async (tx: Tx, model: ApiRootModel): Promise<void> => {
 	return $run(tx, model, runnableMigrations);
 };
 
+export const schedule = async (date: Date, key: string, callback: () => Promise<void>): Promise<void> => {
```

Comment:
TODO: `callback` should instead probably be a `Migration` instead of a generic function.